### PR TITLE
V0.3 import logic testcase

### PR DESCRIPTION
**Pull request by kevinLamKB**
Changelog
- Imported & refactored all remaining logic test cases from V0.1_base
- Fix some JUnit naming conventions in LogicManagerTest
- Contains some changes merged over from logic_date_parsing branch

### CODE REVIEW

**Comment by kevinLamKB** [comment]
+1500 lines of code and only  +3.9% coverage 😢

**Comment by akd13** [comment]
:( maybe check this https://coveralls.io/builds/12312506 to see which files are missing coverage?

File: src/main/java/seedu/multitasky/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -13,7 +20,7 @@
 import seedu.multitasky.model.entry.ReadOnlyEntry;
 import seedu.multitasky.model.tag.Tag;
 
-//@@kevinlamkb A0140633R
+// @@kevinlamkb A0140633R
```
Comment (akd13) [inline src/main/java/seedu/multitasky/logic/parser/AddCommandParser.java]: Change to // @@author

Comment (kevinLamKB) [reply src/main/java/seedu/multitasky/logic/parser/AddCommandParser.java]: ah right, thanks

File: src/main/java/seedu/multitasky/logic/parser/AddCommandParser.java [hunk 2]
```diff
@@ -22,32 +29,108 @@
...
+                                                 PREFIX_TO, PREFIX_TAG);
+        Calendar startDate = Calendar.getInstance();
+        Calendar endDate = Calendar.getInstance();
 
-        if (trimmedArgs.isEmpty()) {
+        // check for no args input
+        if (args.trim().isEmpty()) {
             throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
+                                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
```
Comment (mattheuslee) [inline src/main/java/seedu/multitasky/logic/parser/AddCommandParser.java]: is this from eclipse? doesn't really make sense compared to just having it on the previous line

**Review by mattheuslee — APPROVED** [review]

**Comment by mattheuslee** [comment]
@kevinLamKB heh now it's a 10% jump

**Comment by kevinLamKB** [comment]
@mattheuslee haha wth all i did was go into all of my logic classes and use eclipse autoformat and it jumped 6% 😨

**Comment by mattheuslee** [comment]
@kevinLamKB also cos I just merged in my additional UI and commons tests into V0.3 😃

**Comment by kevinLamKB** [comment]
oh haha now that makes much more sense 😃

**Review by ChuaPingChan — APPROVED** [review]

**Pull request merged**